contracts/interfaces/IPoolParameters.sol

```diff
@@ -115,6 +115,13 @@ interface IPoolParameters {
      */
     function unlimitedApproveTo(address token, address to) external;
 
+    /**
+     * @notice reset token's allowance value to the 'to' address
+     * @param token The ERC20 token address
+     * @param to The address receive the grant
+     */
+    function revokeUnlimitedApprove(address token, address to) external;
```

Comment:
can we merge unlimitedApproveTo and revokeUnlimitedApprove to:

approveToken(address token, address to, uint256 amount)